rule/validator.go

```diff
@@ -95,17 +95,24 @@ func (v *ValidatorDefault) validateAuthorizer(r *Rule) error {
 	return auth.Validate()
 }
 
-func (v *ValidatorDefault) validateMutator(r *Rule) error {
-	if r.Mutator.Handler == "" {
-		return errors.WithStack(herodot.ErrInternalServerError.WithReason(`Value of "mutator.handler" can not be empty.`))
+func (v *ValidatorDefault) validateMutators(r *Rule) error {
+	if len(r.Mutators) == 0 {
+		return errors.WithStack(herodot.ErrInternalServerError.WithReason(`Value of "mutators" must be set and can not be an empty array.`))
 	}
 
-	mutator, err := v.r.PipelineMutator(r.Mutator.Handler)
-	if err != nil {
-		return herodot.ErrInternalServerError.WithReasonf(`Value "%s" of "mutator.handler" is not in list of supported mutators: %v`, r.Mutator.Handler, v.r.AvailablePipelineMutators()).WithTrace(err).WithDebug(err.Error())
+	for k, m := range r.Mutators {
```

Comment:
Maybe we should return a list of wrong mutators instead of returning error for single element?